### PR TITLE
Nailgun Github Action Checks Added

### PR DESCRIPTION
**Pull request by jyejare**
##### Description of changes

Github Actions added to Nailgun that's missing since Travis removed.

### CODE REVIEW

File: .github/workflows/merge_to_master.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    PYCURL_SSL_LIBRARY: openssl
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # python 3.6 is still used by IQE integration pipeline
+        python-version: [3.6, 3.8, 3.9]
```
Comment (JacobCallahan) [inline .github/workflows/merge_to_master.yml:19]: let's add 3.10, perhaps dropping 3.8 or 3.9

File: .github/workflows/pull_requests.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+env:
+    PYCURL_SSL_LIBRARY: openssl
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.8, 3.9]
```
Comment (JacobCallahan) [inline .github/workflows/pull_requests.yml:17]: same suggestion about 3.10

File: setup.py [hunk 3]
```diff
@@ -43,6 +43,8 @@
         'Intended Audience :: Developers',
         'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```
Comment (JacobCallahan) [inline setup.py:47]: assuming 3.10 checks are good we can also add that here

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

File: .github/workflows/merge_to_master.yml (hunk 1 above)
Comment (jyejare) [reply .github/workflows/merge_to_master.yml]: Removed `3.9` and added `3.10` which now checks in even python versions of 6,8 and 10.

File: .github/workflows/pull_requests.yml (hunk 2 above)
Comment (jyejare) [reply .github/workflows/pull_requests.yml]: Removed `3.9` and added `3.10` which now checks in even python versions of 6,8 and 10.

File: setup.py (hunk 3 above)
Comment (jyejare) [reply setup.py]: Removed `3.9` and added `3.10` which now checks in even python versions of 6,8 and 10.

Comment (JacobCallahan) [reply setup.py]: Probably should have kept 3.9, but i don't think it will hurt anything to remove it

**Review by JacobCallahan — APPROVED** [review]

**Review by jameerpathan111 — APPROVED** [review]
LGTM

**Review by swadeley — COMMENTED** [review]
ACK

**Pull request merged**